kmeans.py

```diff
@@ -28,12 +28,25 @@ def add(self, p):
         self.members.append(p)
         return self.recalc_center()
     
-    def recalc_center()->float:
+    def recalc_center(self)->float:
         '''returns eclidean Distance, between the updated centroid to the previous one'''
-        pass
+        coords = () * self.center.dimention
```

Comment:
this returns an empty tuple. If you want an empty tuple the size of the dimension, you can use 
`coords = tuple((None for i in range(self.center.dimention)))`

Reply:
when I do the change you've suggested it doesn't work, you want to try it ?